docs/user/metacat/source/oaipmh.rst

```diff
@@ -1,3 +1,13 @@
+----
+
+.. deprecated:: 3.0.0
+    Harvesting metadata in Metacat through OAI-PMH standard is obsolete.
+
+    While some features remain visible, the old Metacat API has been disabled.
+    All features that depend on the old Metacat API are deprecated.
+
+----
+
 OAI Protocol for Metadata Harvesting
```

Comment:
again - can't we simply remove this altogether?

Reply:
After discussing with Jing, who referenced a discussion with Matt, it seems that there may exist some operators who still want to use the provider service that is part of `OAI-PMH`. I have added an additional deprecated directive to the Harvesting section to further emphasize that Harvesting is deprecated.